### PR TITLE
feat : implemented password update successful modal, as well as testi…

### LINKED ISSUE #172: [Feature]: SETTINGS PAGE - Implement the Password successful modal
### Description

Implement a modal that appears when a user's password update is successful. 
The modal should include a "Continue" button that, when clicked, disables the modal. 
Additionally, users should be able to click outside the modal to disable it.

### Acceptance Criteria

1. The modal should display a message indicating the password update was successful.
2. The modal should include a "Continue" button that disables the modal when clicked.
3. Clicking outside the modal should also disable it.
4. The modal should be styled according to the provided design specifications.
5. The modal should be responsive across all devices (mobile, tablet, desktop).
6. Unit tests should cover the modal's functionality.
7. Integration tests should ensure the modal works correctly within the settings page.

Success Message Box

### Links

https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=10-17157&m=dev

### PR DESCRIPTION
**Pull request by Maiqel1**
…ng and instructions for use in guides

# Description

**Closes #172 **

# Changes proposed

## What were you told to do?
Implement the Password successful modal

## What did you do?
i implemented the password update successful modal, as well as a file for testing. i used tailwind css and shadcn components for styling

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [X] I am making a pull request against the **dev branch** (left side).
- [X] My commit messages styles matches our requested structure.
- [X] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.

# Screenshots/Videos
Create Next App - Google Chrome 7_18_2024 10_20_12 PM

### CODE REVIEW

File: src/app/guides/page.tsx
Comment (Justfemi) [inline src/app/guides/page.tsx:1]: You're not supposed to edit this file

**Review by Justfemi — COMMENTED** [review]
Don't edit the src/guides/page.tsx file
Use [shadcn](https://ui.shadcn.com/docs/components/dialog) for the modal

File: src/app/guides/page.tsx
Comment (prudentbird) [inline src/app/guides/page.tsx:1]: Remove edits on guides/page.tsx

**Review by prudentbird — DISMISSED** [review]

**Review by Justfemi — DISMISSED** [review]
Use [shadcn ](https://ui.shadcn.com/docs/components/dialog)for your modal

**Comment by Maiqel1** [comment]
@Justfemi

**Review by oreoluwa212 — APPROVED** [review]

**Pull request merged**